change/@ni-nimble-components-a3500224-7314-4022-aa82-2f4cf29f7866.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
For your implementation PRs a helpful thing to do is to mention the implementation issue #296 in the PR description. GitHub automatically adds links in both directions so it can make it easier to look a the issue and see all the dev work that's associated with it.